doc/guide/feature-sosreport.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+	"http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="feature-sosreport">
+  <title>SOS Report</title>
+
+  <para>If present on the system Cockpit can use <code>sosreport</code> to collect system configuration and diagnostic information.
+    To perform similar tasks from the command line, use the <code>sosreport</code> command.</para>
```

Comment:
Should we reference what Cockpit does on Atomic systems, also? I think it's nice to know.